src/Azure.Functions.Cli/Helpers/NetworkHelpers.cs

```diff
@@ -33,5 +37,33 @@ public static int GetAvailablePort()
                 listener.Stop();
             }
         }
+
+        public static int GetNextAvailablePort(int startPort)
+        {
```

Comment:
ArgumentExceptin when startPort is either using for system process or upper limit of 65536

Reply:
Done! 👍